packages/integration-tests/src/test-project-setup/setup_dir_as_esm_module.ts

```diff
@@ -25,6 +25,11 @@ export const setupDirAsEsmModule = async (absoluteDirPath: string) => {
     'es2022',
   ];
 
+  await execa('npm', ['install', 'typescript'], {
```

Comment:
`npm` only ?;-)

I think this is another signal to just use create-amplify to boostrap test project.